framework/db/schema/pgsql/CPgsqlSchema.php

```diff
@@ -186,7 +191,7 @@ protected function findColumns($table)
 			$c=$this->createColumn($column);
 			$table->columns[$c->name]=$c;
 
-			if(stripos($column['adsrc'],'nextval')===0 && preg_match('/nextval\([^\']*\'([^\']+)\'[^\)]*\)/i',$column['adsrc'],$matches))
+			if(stripos($column['column_def_value'],'nextval')===0 && preg_match('/nextval\([^\']*\'([^\']+)\'[^\)]*\)/i',$column['column_def_value'],$matches))
```

Comment:
for full BC, the name `adsrc` should be kept using an alias in the query above.

Reply:
hm, if someone has extended the CPgsqlSchema findColumns method, changes in the original method won't affect their new code.

Reply:
`protected function createColumn($column)` depends on the name, so if someone extended that one, it might break.